src/Grpc.NetCore.HttpClient/Internal/ISystemClock.cs

```diff
@@ -0,0 +1,27 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System;
+
+namespace Grpc.NetCore.HttpClient.Internal
+{
+    internal interface ISystemClock
```

Comment:
We keep having this exact interface all around our stack...

Reply:
aspnet/specs is waiting for you Microsoft.Extensions.Clock PR 😋